test/multifile/constant-struct-with-padding/Other.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: true
```

Comment:
nit-pick: Is this `RUN` statement necessary, or did you leave it in as an easy way to get the test to fail (by changing it to `RUN: false`)? Same quesiton for the `RUN` statement in `constant-tuple-with-padding/Other.swift` below.